.github/scripts/render_card.py

```
@@ -0,0 +1,495 @@
+    parts = []
+    if diff["changed_count"]:
+        parts.append(
+            f'<span class="diff-changed">↑ {diff["changed_count"]} updated</span>'
+        )
+    if diff["added_count"]:
+        parts.append(
+            f'<span class="diff-added">+ {diff["added_count"]} added</span>'
+        )
+    if diff["removed_count"]:
+        parts.append(
+            f'<span class="diff-removed">− {diff["removed_count"]} removed</span>'
```

Ruff (0.15.12)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)